src/core/server/saved_objects/migrationsv2/model.test.ts

```diff
@@ -289,39 +290,13 @@ describe('migrations v2 model', () => {
         });
         const newState = model(initState, res) as FatalState;
 
-        expect(newState.controlState).toEqual('SET_SOURCE_WRITE_BLOCK');
+        expect(newState.controlState).toEqual('WAIT_FOR_YELLOW_SOURCE');
         expect(newState).toMatchObject({
-          controlState: 'SET_SOURCE_WRITE_BLOCK',
-          sourceIndex: Option.some('.kibana_7.invalid.0_001'),
-          targetIndex: '.kibana_7.11.0_001',
+          controlState: 'WAIT_FOR_YELLOW_SOURCE',
+          sourceIndex: '.kibana_7.invalid.0_001',
         });
-        // This snapshot asserts that we disable the unknown saved object
-        // type. Because it's mappings are disabled, we also don't copy the
-        // `_meta.migrationMappingPropertyHashes` for the disabled type.
-        expect(newState.targetIndexMappings).toMatchInlineSnapshot(`
-          Object {
-            "_meta": Object {
-              "migrationMappingPropertyHashes": Object {
-                "new_saved_object_type": "4a11183eee21e6fbad864f7a30b39ad0",
-              },
-            },
-            "properties": Object {
-              "disabled_saved_object_type": Object {
-                "dynamic": false,
-                "properties": Object {},
-              },
-              "new_saved_object_type": Object {
-                "properties": Object {
-                  "value": Object {
-                    "type": "text",
-                  },
-                },
-              },
-            },
-          }
-        `);
       });
-      test('INIT -> SET_SOURCE_WRITE_BLOCK when migrating from a v2 migrations index (>= 7.11.0)', () => {
+      test('INIT -> WAIT_FOR_YELLOW_SOURCE when migrating from a v2 migrations index (>= 7.11.0)', () => {
```

Comment:
@rudolf 
the master version of `0013_saved_object_migrations.md` doesn't contain description for every implementation step. I could add `WAIT_FOR_YELLOW_SOURCE` to https://github.com/elastic/kibana/pull/93002

Reply:
yes, it would be great to document it there.